themes/default/data/registry/packages/scaleway.yaml

```
@@ -1,14 +1,14 @@
-category: Infrastructure
+category: Cloud
 component: false
 description: A Pulumi package for creating and managing scaleway cloud resources.
 featured: false
-logo_url: "https://raw.githubusercontent.com/jaxxstorm/pulumi-scaleway/master/assets/scaleway-svgrepo-com.svg"
+logo_url: ""
 name: scaleway
 native: false
 package_status: public_preview
-publisher: jaxxstorm
+publisher: Lee Briggs
 repo_url: https://github.com/jaxxstorm/pulumi-scaleway
 schema_file_path: /provider/cmd/pulumi-resource-scaleway/schema.json
 title: scaleway
```

Nit, this should be title-capped as well.

```suggestion
title: Scaleway
```